src/utils/transaction/insertTransaction.ts

```diff
@@ -70,5 +72,13 @@ export const insertTransaction = async (
   });
   reportUsage([{ action: "queue_tx", input: queuedTransaction }]);
 
+  recordMetrics({
+    event: "transaction_queued",
+    params: {
+      chainId: insertedTransaction.chainId.toString(),
+      walletAddress: getAddress(insertedTransaction.from),
```

Comment:
Nit: use `queuedTransaction` here since it's more accurate here (it's recording a "queued transaction" event). The `queuedTransaction.from` is already checksum case too.